hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/endpoint/ObjectEndpoint.java

```diff
@@ -882,6 +888,20 @@ private Response createMultipartKey(OzoneVolume volume, String bucket,
               .createMultipartKey(ozoneBucket, key, length, partNumber,
                   uploadID, chunkSize, (DigestInputStream) body);
         }
+
+        copyHeader = headers.getHeaderString(COPY_SOURCE_HEADER);
+        if (copyHeader != null) {
+          String range =
```

Comment:
Shall we add this COPY_SOURCE_HEADER_RANGE check in the put() L253 too?

Reply:
I checked the CopyObject document, it looks like range copy is not supported.

https://docs.aws.amazon.com/AmazonS3/latest/API/API_CopyObject.html